src/main/java/com/spartronics4915/frc2019/vision/GoalTracker.java

```diff
@@ -0,0 +1,258 @@
+package com.spartronics4915.frc2019.vision;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.Iterator;
+import java.util.List;
+
+import com.spartronics4915.frc2019.Constants;
+
+import com.spartronics4915.lib.geometry.Pose2d;
+
+import com.spartronics4915.lib.geometry.Translation2d;
+import com.spartronics4915.lib.geometry.Rotation2d;
+import com.spartronics4915.lib.util.InterpolatingTreeMap;
+import com.spartronics4915.lib.util.InterpolatingDouble;
+
+/**
+ * This is used in the event that multiple goals are detected to judge all goals
+ * based on timestamp, stability, and continuation of previous goals (i.e. if a
+ * goal was detected earlier and has changed locations). This allows the robot
+ * to make consistent decisions about which goal to aim at and to smooth out
+ * jitter from vibration of the camera.
+ * 
+ * @see GoalTrack.java
+ */
+public class GoalTracker
+{
+
+    /**
+     * Track reports contain all of the relevant information about a given goal
+     * track.
+     */
+
+    // TODO: Find a better home for these varibles
+
+    private Rotation2d camera_pitch_correction_;
```

Comment:
convert these to our mMemberVar rep?

Reply:
Btw, `F2` renames variables in VSCode, if you didn't know that already.